docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# docker-compose.yml
+
+version: '3.10'
+services:
+  flask:
+    image: flask
+#    build: .
+    ports:
+      - "5000:5000"
+    volumes:
+      - ./flaskapp:/app
+    environment:
+      - FLASK_ENV=development
+  mariadb:
```

Comment:
Does mariadb need it's own volume to persist as well or does it just work? I haven't ran this yet so I'm not sure.

Reply:
it should just work, i just translated the script to work with docker compose, and the script works